lively.project/project.js

```diff
@@ -152,27 +152,22 @@ export class Project {
 
     const checkLivelyCompatability = await loadedProject.bindAgainstCurrentLivelyVersion(loadedProject.config.lively.boundLivelyVersion);
 
-    let loadingCanceled = false;
     switch (checkLivelyCompatability) {
       case 'CANCELED':
       case 'OUTDATED': {
-        loadingCanceled = true;
-        await $world.inform('Due to incompatible lively versions, you cannot open this project. This session will now close. This probably means that your version is older than the one the project requires.');
-        window.location.href = (await Project.systemInterface.getConfig().baseURL);
+        await $world.inform('The required lively version of this project conflicts with the running one.', { additionalText: 'You can proceed with OK, but be aware that some expected behaviour might differ or not work.' });
```

Comment:
<img width="866" alt="formatting-issue" src="https://github.com/LivelyKernel/lively.next/assets/1296388/fe9dfe27-587a-4647-b832-d337ff124df4">
Please fix the formatting of the additional text. This looks to much out of place for my taste.